cluster/ci/monitoring/mixins/config.libsonnet

```diff
@@ -21,6 +21,11 @@
         channel: '#build-cop-alerts',
         notify: 'build-cop',
       },
+      'openshift-library': {
+        team: 'openshift-library',
+        channel: '#tbd',
+        notify: 'tbd',
```

Comment:
@bparees 
Could you please provide the channel name and the team/person name in slack when `periodic-openshift-library-import` fails?

Reply:
`#forum-devex`
`@devex`

thanks!

@coreydaley @adambkaplan fyi.

Reply:
Done.